src/index.js

```diff
@@ -100,17 +112,16 @@ let theUrl
  * @param {string} manifest
  * @param {false} html
  * @param {object} [opts={}]
- * @returns {Promise<AnalysisReport>}
+ * @returns {Promise<import('@trustification/exhort-api-spec/model/v4/AnalysisReport').AnalysisReport>}
  * @throws {Error}
  */
 
 /**
  * Get stack analysis report for a manifest file.
- * @overload
```

Comment:
This `@overload` needs to stay. Without it, we/consumers of this API can only pass types that are known to be `true` or `false` rather than a boolean type that could be either.
Example error:
```
No overload matches this call.
  Overload 1 of 2, '(manifest: string, html: true, opts?: object): Promise<string>', gave the following error.
    Argument of type 'boolean' is not assignable to parameter of type 'true'.
  Overload 2 of 2, '(manifest: string, html: false, opts?: object): Promise<AnalysisReport>', gave the following error.
    Argument of type 'boolean' is not assignable to parameter of type 'false'.
```

Reply:
updated